src/refactor_nrepl/artifacts.clj

```diff
@@ -36,18 +43,7 @@
          (map edn/read-string))
     (catch Exception _
       ;; In the event clojars is down just return an empty vector. See #136.
```

Comment:
thanks for catching that. actually it shoud be a vector as it was before, so the comment is ok but vector (instead of map) should be returned in this case.